spec/marks_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+require_relative '../lib/marks'
+
+RSpec.describe Marks do
+
+  it "returns 'X'" do
+    expect(Marks.switch_mark("O")).to eq("X")
+  end
+
+  it "returns 'O'" do
+    expect(Marks.switch_mark("X")).to eq("O")
+  end
+
```

Comment:
two empty lines - inconsistent with the rest of your files where you only have one empty line.